src/components/GameItem.js

```diff
@@ -4,26 +4,49 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  staticpropTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+  constructor(props) {
+    super();
+    this.state = {
+      clicked: false,
+    }
+  }
+
+  onIconClick = () => {
+    this.props.updateScoreCallback(this.props.index);
+    this.setState({
+      clicked: true,
+    });
+  }
+
   render() {
+    let clickStyle = undefined;
+    if (this.state.clicked) {
+      if (this.props.type === 'litter') {
+        clickStyle = 'spotted-litter';
+      } else {
+        clickStyle = 'spotted-nature';
+      }
+    }
+
     const itemStyle = {
       bottom: `${this.props.height}px`, // use props.height to offset from the bottom of screen
       zIndex: this.props.layer, // use props.layer to set z-index, so we display ontop of background
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    const icon = ItemIcons[this.props.type];
 
     return (
-      <div className="game-item" style={itemStyle}>
-        <img src={icon} alt="Item" className="icon-item"></img>
+      <div className={`game-item ${clickStyle}`} style={itemStyle}>
+        <img onClick={ this.onIconClick} src={icon} alt="Item" className="icon-item"></img>
```

Comment:
You may want to place the `onClick` on the `<div>` to make it easier to click on.  (Though this will allow clicking on the same piece of litter more than once.)